server/db/index.js

```diff
@@ -0,0 +1,30 @@
+var ENV = 'development';
```

Comment:
This will eventually need to handle multiple environments, right? Might be worth commenting to indicate that there's a to do here, but is fine for now